website/package.json

```diff
@@ -57,12 +57,13 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-lite-youtube-embed": "^2.2.2",
-    "react-medium-image-zoom": "^4.4.0",
+    "react-medium-image-zoom": "^4.4.3",
     "react-popper": "^2.3.0",
     "rehype-katex": "^6.0.2",
     "remark-math": "^3.0.1",
     "swc-loader": "^0.2.1",
     "unist-util-visit": "^2.0.3",
+    "webpack": "^5.72.1",
```

Comment:
why do we need it to be added now?

Reply:
To fix a peer dependency warning, because we installed raw-loader. The warning had always been there, but I just got around to fix it now, since most lerna warnings are gone, making the rest more prominent.